internal/controller/clusterorder_controller.go

```diff
@@ -81,19 +76,25 @@ func newClusterReference() *v1alpha1.ClusterOrderClusterReferenceType {
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
 func (r *ClusterOrderReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = ctrllog.FromContext(ctx)
+	log := ctrllog.FromContext(ctx)
 
 	instance := &v1alpha1.ClusterOrder{}
 	err := r.Client.Get(ctx, req.NamespacedName, instance)
 	if err != nil {
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	log.Info(fmt.Sprintf("Start reconcile for %s", instance.GetName()))
```

Comment:
there's no `log.Infof` to avoid the nested `fmt.Sprintf`?

Reply:
Not in [this log package](https://pkg.go.dev/github.com/go-logr/logr#Logger).

Reply:
https://github.com/go-logr/logr?tab=readme-ov-file#but-i-really-want-to-use-a-format-string

looks like this logger wants to push another way to log things